.pre-commit-config.yaml

```
@@ -21,8 +22,14 @@ repos:
         id: prettier
   - repo: local
     hooks:
-      - id: golangci-lint
-        name: golangci-lint
-        entry: golangci-lint run
-        language: system
-        types: [go]
+      - id: custom-pre-commit
+        name: Custom pre-commit hook
+        entry: ./githooks/pre-commit
+        language: script
+        stages: [commit]
```

```suggestion
        stages: [pre-commit]
```
As per the documentation: https://pre-commit.com/#confining-hooks-to-run-at-certain-stages